owasm/src/ext/crypto/coins.rs

```diff
@@ -2,7 +2,15 @@ use serde::{Deserialize, Serialize};
 
 #[derive(Debug, Serialize, Deserialize, PartialEq)]
 pub enum Coins {
+    ADA,
```

Comment:
@taobun @prin-r Important to note that doing this breaks backward compatibility with old scripts that use older version of owasm because the enum underlying values are mixed up. 

Better way to do it (if this is already live) is to add enum values after the existing ones.